compose/ui/ui-graphics/src/commonMain/kotlin/androidx/compose/ui/graphics/layer/GraphicsLayer.kt

```diff
@@ -236,8 +236,6 @@ expect class GraphicsLayer {
      */
     fun setPathOutline(path: Path)
 
-    // TODO Fix default values on Wasm
```

Comment:
What is the context here? YT link?

Reply:
At the time I wrote this, there was this code:
```
fun setRoundRectOutline(
        topLeft: Offset = Offset.Zero,
        size: Size = UnspecifiedSize,
        cornerRadius: Float = 0f,
    )

companion object {
    val UnspecifiedSize
}
```

it failed on wasm, probably because of some initialization order error. Now we have this:
```
fun setRoundRectOutline(
        topLeft: Offset = Offset.Zero,
        size: Size = Size.Unspecified,
        cornerRadius: Float = 0f,
    )
```
and it doesn't fail anymore